patches/wxWidgets/textctrl_osx_return.patch

```diff
@@ -0,0 +1,18 @@
+diff --git src/osx/textctrl_osx.cpp src/osx/textctrl_osx.cpp
+index 1ef8afa420..452d4272cf 100644
+--- src/osx/textctrl_osx.cpp
++++ src/osx/textctrl_osx.cpp
+@@ -342,6 +342,13 @@ void wxTextCtrl::OnDropFiles(wxDropFilesEvent& event)
+ 
+ void wxTextCtrl::OnKeyDown(wxKeyEvent& event)
+ {
++    int key = event.GetKeyCode();
++    if ( key == WXK_RETURN || key == WXK_NUMPAD_ENTER ) {
++        // HACK: Forward these keys to the OnChar handler
++        OnChar(event);
++        return;
```

Comment:
Does the event need to be skipped, too, i.e. do you need to call event.Skip() to avoid further processing?

Reply:
OnChar will call .Skip() if it didn't consume the event, so I think this is OK.

https://github.com/wxWidgets/wxWidgets/blob/v3.1.1/src/osx/textctrl_osx.cpp#L473